extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -60,6 +60,9 @@ public class ParquetTableLocation extends AbstractTableLocation {
 
     // Access to all the following variables must be guarded by initialize()
     // -----------------------------------------------------------------------
+    private ParquetFileReader parquetFileReader;
+    private int[] rowGroupIndices;
```

Comment:
Interesting to note that these variables are only needed to support further delayed rowGroupReaders. Can you move to the bottom of the comment block?

I wonder if we should null them out right before setting rowGroupReaders.